cmd/cli/mesh_list.go

```diff
@@ -115,3 +138,38 @@ func getMeshNames(clientSet kubernetes.Interface) mapset.Set {
 
 	return meshList
 }
+
+func getSupportedSmiForControllerPod(pod string, namespace string, cmd *meshListCmd) (map[string]string, error) {
+	dialer, err := k8s.DialerToPod(cmd.config, cmd.clientSet, pod, namespace)
+	if err != nil {
+		return nil, err
+	}
+
+	portForwarder, err := k8s.NewPortForwarder(dialer, fmt.Sprintf("%d:%d", cmd.localPort, constants.OSMHTTPServerPort))
+	if err != nil {
+		return nil, errors.Errorf("Error setting up port forwarding: %s", err)
+	}
+
+	var smiSupported map[string]string
+
+	err = portForwarder.Start(func(pf *k8s.PortForwarder) error {
+		defer pf.Stop()
+		url := fmt.Sprintf("http://localhost:%d/%s", cmd.localPort, "smi/version")
```

Comment:
would be nice if we had a shared define for "smi/version", so if it changes in time, all references get updated.

Reply:
Thanks @eduser25. Implemented it.
![image](https://user-images.githubusercontent.com/13926417/116328393-c3a43000-a77d-11eb-9eb3-ae5bb2a845d1.png)